README.md

```diff
@@ -69,7 +69,8 @@ use winit::event::{Event, WindowEvent};
 use winit::event_loop::{ControlFlow, EventLoop};
 use winit::window::Window;
 
-include!("../examples/utils/winit_app.rs");
+#[path = "../examples/utils/winit_app.rs"]
```

Comment:
I would have gone further and not used `path` at all.
This can be done by moving the file directly to `examples/winit_app.rs` and excluding it as an auto discoverable example with `[[example]]` in `Cargo.toml`.

Just a suggestion, can be done in a follow-up as well or not at all if ppl don't like it.

Reply:
@daxpedda this was handled in https://github.com/rust-windowing/softbuffer/pull/227, I don't remember if I asked whether to add a `mod.rs` instead.